### PR TITLE
Update 2020-07-09-product-features-methodology-at-prestashop.md

### PR DESCRIPTION
**Pull request by ttoine**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. Every detail helps: What are you changing? Why?
| Type?         | bug fix / improvement / new feature / refacto
| BC breaks?    | yes / no
| Deprecations? | yes / no
| Fixed ticket? | Fixes {paste the issue here}.
| How to test?  | Please indicate how to best verify that this PR is correct.

### CODE REVIEW

**Review by LouiseBonnard — APPROVED** [review]

File: news/_posts/2020-07-09-product-features-methodology-at-prestashop.md [hunk 1]
```diff
@@ -80,8 +80,6 @@ Lastly, once you know what to focus on, and what you should work on next, it is
 
 All the product development process at PrestaShop is summarized in the following chart.
 
-![Product development steps at PrestaShop](/assets/images/2020/05/product-development-steps.png)
-
```
Comment (LouiseBonnard) [inline news/_posts/2020-07-09-product-features-methodology-at-prestashop.md]: Delete all lines please, there is no following chart in the end, thank you very much!

Comment (LouiseBonnard) [reply news/_posts/2020-07-09-product-features-methodology-at-prestashop.md]: From 81 to 84.

**Comment by ttoine** [comment]
@LouiseBonnard can you confirm I deleted the good lines?

**Comment by LouiseBonnard** [comment]
@ttoine, now it's perfect, thanks a lot!

**Pull request merged**